demisto_sdk/commands/update_release_notes/update_rn.py

```diff
@@ -129,6 +133,19 @@ def _does_pack_metadata_exist(self):
 
         return True
 
+    def get_master_version(self):
+        """
+        Get the current version from origin/master if available, otherwise return '0.0.0'
+        """
+        master_current_version = '0.0.0'
+        master_metadata = get_remote_file(self.metadata_path)
```

Comment:
what happens if the file does not exist on master - maybe we need try/except here?

Reply:
You are right, I added an try exception, with an error saying that master isnt reachable so the version will be taken from the local metadata file.